src/EnergyPlus/ZoneAirLoopEquipmentManager.cc

```diff
@@ -567,10 +509,8 @@ namespace ZoneAirLoopEquipmentManager {
         // This subroutine is left for Module format consistency -- not needed in this module.
 
         // Do the Begin Simulation initializations
-        if (InitAirDistUnitsFlag) {
-            EachOnceFlag.allocate(NumAirDistUnits);
-            EachOnceFlag = true;
-            InitAirDistUnitsFlag = false;
+        if (!InitAirDistUnitsFlag) {
```

Comment:
Repurpose `InitAirDistUnitsFlag` to avoid checking `EachOnceFlag(AirDistUnitNum)` throughout the simulation.